cmd/docs.go

```diff
@@ -95,4 +122,5 @@ var docsCmd = &cobra.Command{
 
 func init() {
 	RootCmd.AddCommand(docsCmd)
+	docsCmd.Flags().UintP("width", "w", 0, "Set word-wrap width (0 disables word wrapping). Influences output formatting, particularly for tables.")
```

Comment:
Seems more appropriate for `docs.width` setting in `atmos.yaml`

Reply:
Would you envision it under the `settings` block or a separate `docs` structure?

```yaml
docs:
  width: 140
  # And eventually...
  pagination: (false | true)
```

Reply:
Should it be max-width? Since width is otherwise dynamic?

Reply:
Or if width is "auto" or undefined it's responsive, and otherwise constant.

Reply:
Just added the blocks to the `atmos.yaml`!
```yaml
settings:
  docs:
    max-width: 140
```

This is my first stab at this kind of change so just let me know of any other areas you'd like updated! I tested it with our configuration and it works as intended